deno/send_message/README.md

```diff
@@ -0,0 +1,101 @@
+# SendMessage() 
+
+A sample Deno Cloud Function sending a message using a specific channel to a receiver
+
+Supported channels are `SMS`, `EMAIL` ,`DISCORD` and `TWITTER`.
+
+
+_Example function payloads:_
+
+```json
+// Sending SMS
+{
+    "type":"SMS","receiver":"+123456789","message":"Programming is fun!"
+}
+// Sending Email
+{
+    "type":"EMAIL","receiver":"hello@example.com",
+    "message":"Programming is fun!","subject":"Programming is funny!"
+}
+// Sending message through Discord webhook 
+{
+    "type":"DISCORD","receiver":"","message":"Hi"
+}
+// Sending a tweet
+{
+    "type":"TWITTER","receiver":"","message":"Programming is fun!"
+}
+```
+
+
+_Successful function response:_
+
+
+```json
+{ "success" : true }
+```
+
+_Error function response:_
+
+
+```json
+{
+	"success": false,
+	"error": "Failed to send message, check the webhook URL"
+}
+```
+
+## 📝 Environment Variables
+
+List of environment variables used by this cloud function:
```

Comment:
Let's call this variables instead of environment variables.

Reply:
All examples use "environment variables" as far as I can see, both new and old. 🤔 Changing this to "variables" would remove the parity between examples.

Reply:
ok then what should I do?

Reply:
Let's do "variables". We're currently cleaning up some other "environment" leftovers in other places.